docs/index.md

```diff
@@ -209,14 +209,14 @@ If that sounds like something you're interested in, I'd love to have you along o
 
 ## License
 
-Foam is licensed under the [MIT license](license).
+Foam is licensed under the [MIT license](LICENSE.txt).
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[graph-visualisation]: features/graph-visualisation.md "Graph Visualisation"
-[backlinking]: features/backlinking.md "Backlinking"
-[recommended-extensions]: recommended-extensions.md "Recommended Extensions"
-[recipes]: recipes/recipes.md "Recipes"
-[frequently-asked-questions]: frequently-asked-questions.md "Frequently Asked Questions"
-[principles]: principles.md "Principles"
-[contribution-guide]: contribution-guide.md "Contribution Guide"
+[graph-visualisation]: features/graph-visualisation "Graph Visualisation"
+[backlinking]: features/backlinking "Backlinking"
+[recommended-extensions]: recommended-extensions "Recommended Extensions"
+[recipes]: recipes/recipes "Recipes"
+[frequently-asked-questions]: frequently-asked-questions "Frequently Asked Questions"
+[principles]: principles "Principles"
+[contribution-guide]: contribution-guide "Contribution Guide"
```

Comment:
I'm confused with the removal of the `.md`, as far as I know this would break links on GitHub, maybe not on the site though. @riccardoferretti what is your opinion?

Reply:
The code block was autogenerated. The removal of .md was not done by me. 

```
[//begin]: # "Autogenerated link references for markdown compatibility"
[graph-visualisation]: features/graph-visualisation.md "Graph Visualisation"
[backlinking]: features/backlinking.md "Backlinking"
[recommended-extensions]: recommended-extensions.md "Recommended Extensions"
[recipes]: recipes/recipes.md "Recipes"
[frequently-asked-questions]: frequently-asked-questions.md "Frequently Asked Questions"
[principles]: principles.md "Principles"
[contribution-guide]: contribution-guide.md "Contribution Guide"
[//end]: # "Autogenerated link references"
```

Reply:
this is due to the configuration for the wikilink definition generation, it needs to be set to `withExtension`

Reply:
@riccardoferretti I guess I understood what you said. In VS Code extension, I modified the property as follows `    "foam.edit.linkReferenceDefinitions": "withExtensions"` .
This change retained the extension in the autogenerated part. Is this right?